src/test/java/gov/healthit/chpl/aqa/pageObjects/ListingDetailsPage.java

```diff
@@ -14,35 +14,23 @@
 
     private WebDriver driver;
     private static WebElement element = null;
-
-    public static void sedDetailsAccordionOpen(final WebDriver driver) {
-        int delay = 20;
-        WebDriverWait wait = new WebDriverWait(driver, delay);
-        wait.until(ExpectedConditions.presenceOfElementLocated(By.id("details-sed")));
-
-        // to close certification criteria accordion that's open by default
-        certificationCriteriaAccordion(driver);
-
-        //to open SED accordion
-        sedAccordion(driver).click();
-    }
-
-    public static WebElement listingName(final WebDriver driver) {
-        element = driver.findElement(By.id("listingName"));
+    
+    public static WebElement acceessibilityStandardText(final WebDriver driver) {
+        element = driver.findElement(By.xpath("//*[@id=\"panel-certification-criteria\"]/div[54]/div/div/div/table/tbody/tr[3]/td[2]/ul/li[1]"));
         return element;
     }
-
-    public static WebElement editCertifiedProductLink(final WebDriver driver) {
-        element = driver.findElement(By.linkText("Edit Certified Product"));
-        element.click();
+    
+    public static WebElement additionalInfoAccordion(final WebDriver driver) {
+        element = driver.findElement(By.id("details-additional-information"));
         return element;
     }
-
-    public static WebElement sEDCriteriaTable(final WebDriver driver) {
-        element = driver.findElement(By.xpath("//*[@id=\"sed-ucd-processes-table\"]"));
+    
+    public static WebElement certificationCriteriaAccordion(final WebDriver driver) {
+        element = driver.findElement(By.id("details-certification-criteria"));
+        element.click();
```

Comment:
Why does this execute the click? It would expand / contract the accordion element, right? If the method is only to return teh element, then I don't think it should do things with the element.